src/odemis/driver/technolution.py

```diff
@@ -53,7 +53,14 @@
 INT16 = numpy.iinfo(numpy.int16)
 VOLT_RANGE = (-10, 10)
 DATA_CONTENT_TO_ASM = {"empty": None, "thumbnail": True, "full": False}
-
+RUNNING = "RUNNING"
+FINISHED = "FINISHED"
+FAILED = "FAILED"
+ASM_INSTALLATION_STATE_MAP = {
+    RUNNING : "installation in progress",
+    FINISHED: "last installation successful",
+    FAILED  : "last installation failed"
+}
```

Comment:
It's not used anywhere, and I don't think it's really helpful to have it there as a dict. So I'd suggest to remove this. What you could do is to use this "user-friendly" messages as the content of the main constants (eg, `RUNNING= "installation in progress"`) This way, the client could just show the status to the user as-is.

Reply:
@Sabrina85 I think we had a discussion on that the dict should be implemented. As far as I can remember there is no strong reason not to remove the dict as Eric suggested but maybe I missed something.
Do you think it is fine to do as Eric suggested?

Reply:
I don't remember the details completely. I just checked in the specs, that instead of returning `RUNNING` in the `getStateInstallation()` call, we thought to return the user friendly message from the dict. I just saw that I missed that in the review. The question is what will we do with the return value `RUNNING` as it is now implemented. That variable itself is not defined I think and thus will raise an error.

The idea was that the `getStateInstallation()` method would return the following:
`return ASM_INSTALLATION_STATE_MAP["RUNNING"]`

And then use this message somewhere.

But maybe this is one level too low? The problem with this method was, that it is a wrapper for two calls to the ASM, but we are only interested in 3 answers (installation running, installation failed, installation successful). Only returning boolean does not work obviously. That's why we came up with the dict. Do you have a good idea how we could solve this?

Reply:
If you return the constant and that the constant value is the user-friendly message, you get the best of both world ;-)